doc/manual/en/index.rst

```diff
@@ -8,23 +8,78 @@ It can access the KNX-Bus directly (via eibd/knxd) or use
 `OpenHAB <http://www.openhab.org/>`__ as a backend. The current version
 can be found on `Github <http://github.com/CometVisu/CometVisu>`__.
 
-.. toctree::
-    :hidden:
-    :glob:
+.. _system-requirements:
 
-    install/index
+System Requirements
+-------------------
+
+Operation by the user takes place via a web browser. The
+Most current browsers are supported.
+
+By specifying :doc:`URL Parameter <config/url-params>` can
+you additionally affect the CometVisu, for example by loading other configs
+or designs, as well forcing rendering for mobile devices such as mobile
+devices (e.g. smartphones).
+
+
+Operating CometVisu
+-------------------
+
+The main menu is in the footer and normally contains
+different links:
+
+-   "**CometVisu.org**" link to the projects homepage
+-   "**Reload**" to reload the current page
 
-Installation
-------------
+    .. caution::
+        At editing mode all unsaved changes will be lost without warning!
```

Comment:
Die Warnung können wir rausnehmen (auch in DE), da die für den alten Editor gegolten hat, der schon länger nicht mehr mit dabei ist